src/main/java/frc/trigon/robot/subsystems/climber/ClimberIO.java

```diff
@@ -0,0 +1,41 @@
+package frc.trigon.robot.subsystems.climber;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.subsystems.climber.placeholderclimber.PlaceholderClimberIO;
+import frc.trigon.robot.subsystems.climber.simulationclimber.SimulationClimberIO;
+import org.littletonrobotics.junction.AutoLog;
+
+public class ClimberIO {
+    static ClimberIO generateIO() {
+        if (RobotConstants.IS_REPLAY)
+            return new ClimberIO();
+        if (RobotConstants.ROBOT_TYPE == RobotConstants.RobotType.PLACEHOLDER)
+            return new PlaceholderClimberIO();
+        return new SimulationClimberIO();
+    }
+
+    protected void updateInputs(ClimberInputsAutoLogged inputs) {
+    }
+
+    protected void setPosition(Rotation2d averagePosition, Rotation2d differentialPosition) {
+    }
```

Comment:
make this only accept a target positon in meters, I don't understand why the average position is a rotation2d. Also, `setTargetPosition`